plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/tasks/MySQLDatabaseExportHandler.java

```diff
@@ -172,7 +172,7 @@ protected void startProcessHandler(
 
 
     static class DumpFilterJob extends DumpJob {
-        private final Pattern DEFINER_PATTER = Pattern.compile("DEFINER\\s*=\\s*`[^*]*`@`[0-9a-z\\-_\\.%]*`", Pattern.CASE_INSENSITIVE);
+        private final Pattern DEFINER_PATTER = Pattern.compile("DEFINER\\s*=\\s*`[^*]*`@`([0-9a-z\\-_\\.%]*|\\[[0-9a-z\\-_.%/]*])`", Pattern.CASE_INSENSITIVE);
```

Comment:
I wonder if we actually care about the contents of the definer. Could just look for `` DEFINER\s*=\s*`.*?`@`.*?` ``, no?